src/routes/jira/connect/enterprise/jira-connect-enterprise-post.ts

```diff
@@ -33,9 +35,9 @@ export const gheServerUrlErrors: GheServerUrlErrors = {
 		errorCode: "GHE_SERVER_BAD_GATEWAY",
 		message: "Bad gateway"
 	},
-	default: {
-		errorCode: "GHE_ERROR_DEFAULT",
-		message: "Something went wrong"
+	ECONNABORTED: {
```

Comment:
Do we want to get rid of the default? What will be displayed when an user hits an error we haven't mapped? I think we should leave the default so we still catch unknown errors, and continue to update the error mapping as more errors surface.